site/index.ejs

```diff
@@ -39,8 +39,86 @@
         ga('create', 'UA-16654919-1', 'auto');
         ga('send', 'pageview');
     </script><% } %>
+    <style>
+      .accessibilityDay {
+        filter: grayscale(100%);
```

Comment:
Could we get the following browser specific filters added to this please?

        -webkit-filter: grayscale(100%);
        -moz-filter: grayscale(100%);
        -ms-filter: grayscale(100%);
        -o-filter: grayscale(100%);

I can't seem to find a simple solution for IE10 and IE11. There are some convoluted workarounds out there, but they all looked a little bit complex compared to how these have been applied